modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml

```diff
@@ -14,16 +14,19 @@ bundle: image
 mode: localgov_newsroom_teaser
```

Comment:
I _think_ this file should be in the localgov_news repo. I'm surprised that we seem to have this image style in localgov_core

Reply:
Thanks @markconroy is there a way for me to establish which repo this file should be in, as it is in the localgov_core repo as of now?

Reply:
Hey both - it's been in this repo for 3 years, so lets create another issue if we're going to remove it.

Reply:
A new issue for relocating this file has been created [here](https://github.com/localgovdrupal/localgov_core/issues/167)